wwwroot/index.html

```diff
@@ -507,10 +765,10 @@ <h4 class="card-title">Max Hillebrand</h4>
 		<footer>
 			Legal notice:
 			<a class="go-to ml-1 mr-1"
-				href="https://zksnacks.com/wp-content/uploads/2018/05/zkSNACKs_Privacy-Policy_vfinal.pdf"
+				href="https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Legal/Assets/LegalDocuments.txt"
```

Comment:
@bharmat @hgergoil old file was missing. Can we use the txt file from WalletWasabi repo?

Reply:
No, @hgergoil is going to make a separate PP for the website.

Reply:
#10